### ISSUE TITLE
The Whisper Hybrid encoder model with dynamic quantization functions properly, but it fails when using a full int8 model with post-training quantization.

### ISSUE THREAD

**Issue by pkgoogle** [google-ai-edge/LiteRT#136]
Original Issue: https://github.com/tensorflow/tensorflow/issues/59716
Opening on behalf of @nyadla-sys

### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installation (pip package or built from source):
- TensorFlow library (version, if pip package or github SHA, if built from source):

### 2. Code

Refer the below colab to reproduce the issue ..
https://colab.research.google.com/drive/1S_3bVlwRZkMaYvvKwtPfWlyXQLS0Bvxa?usp=sharing

```
(You can paste links or attach files by dragging & dropping them below)
- Provide links to your updated versions of the above two colab notebooks.
- Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model.
```

#### Option B: Paste your code here or provide a link to a custom end-to-end colab

```
(You can paste links or attach files by dragging & dropping them below)
- Include code to invoke the TFLite Converter Python API and the errors.
- Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model.
```

### 3. Failure after conversion
If the conversion is successful, but the generated model is wrong, then state what is wrong:

- Model produces wrong results and/or has lesser accuracy.
- Model produces correct results, but it is slower than expected.

### 4. (optional) RNN conversion support
If converting TF RNN to TFLite fused RNN ops, please prefix [RNN] in the title.

### 5. (optional) Any other info / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by gaikwadrahul8** [comment]
This issue originally reported by @nyadla-sys has been moved to this dedicated repository for [LiteRT](https://github.com/google-ai-edge/LiteRT/) to enhance issue tracking and prioritization. To ensure continuity, we have created this new issue on your behalf.

We appreciate your understanding and look forward to your continued involvement.

**Comment by pkgoogle** [comment]
Hi @nyadla-sys, can you try with [AI-Edge-Torch](https://github.com/google-ai-edge/ai-edge-torch)? Let us know how that works out.
